### PR TITLE
Fix alpine image to 3.19.0

### PR DESCRIPTION
**Pull request by prakrit55**
## Which problem is this PR solving?
fixes: #5450

## Description of the changes
- changed alpine image tag from 3.16 to 3.19.0

## How was this change tested?
- 

## Checklist
- [ ] I have read https://github.com/jaegertracing/jaeger/blob/master/CONTRIBUTING_GUIDELINES.md
- [ ] I have signed all commits
- [ ] I have added unit tests for the new functionality
- [ ] I have run lint and test steps successfully
  - for `jaeger`: `make lint test`
  - for `jaeger-ui`: `yarn lint` and `yarn test`

### CODE REVIEW

File: docker/Makefile [hunk 1]
```diff
@@ -1,6 +1,3 @@
-VERSION := 1.0.0
```
Comment (yurishkuro) [inline docker/Makefile:1]: version is not an image version. By 3 I was referring to GOLANG_IMAGE

**Review by yurishkuro — APPROVED** [review]

**Comment by yurishkuro** [comment]
Probably need to make a fix to `./scripts/check-go-version.sh` to pass the linter. The script is currently checking the Makefile:
```
check docker/Makefile "^.*golang:$version_regex" "$go_latest_version"
```
Instead it should be pointed to check the Docker file `docker/base/Dockerfile`

**Comment by prakrit55** [comment]
> Probably need to make a fix to `./scripts/check-go-version.sh` to pass the linter. The script is currently checking the Makefile:
> 
> ```
> check docker/Makefile "^.*golang:$version_regex" "$go_latest_version"
> ```
> 
> Instead it should be pointed to check the Docker file `docker/base/Dockerfile`

I think it should point to the Dockerfile in the debug

**Review by yurishkuro — APPROVED** [review]

**Comment by yurishkuro** [comment]
Thanks!

File: docker/base/Dockerfile [hunk 2]
```diff
@@ -1,5 +1,5 @@
-ARG cert_image
-ARG root_image
+ARG cert_image=alpine:3.19.0
+ARG root_image=alpine:3.19.0
```
Comment (yurishkuro) [inline docker/base/Dockerfile:2]: dependabot doesn't seem to be able to handle this, it did not raise a PR to upgrade to 3.19.1

I think we need to remove these args altogether and define versions directly in the `FROM` directive. That dependabot should definitely support.

**Pull request merged**